exercises/practice/luhn-trait/.meta/config.json

```diff
@@ -32,5 +32,8 @@
     "example": [
       ".meta/example.rs"
     ]
+  },
+  "custom": {
+    "allowed-to-not-compile": "Stub doesn't compile because the custom trait is not implemented for the every type in the test suite. This exercise is an introduction to implementing custom traits for the primitives and fully implementing the trait for the test suite reduces learning."
```

Comment:
"the every"

Reply:
```suggestion
    "allowed-to-not-compile": "Stub doesn't compile because the custom trait is not implemented for every type in the test suite. This exercise is an introduction to implementing custom traits for the primitives and fully implementing the trait for the test suite reduces learning."
```